docs/content/guides/recipes/howto-author-recipes/index.md

```diff
@@ -107,7 +107,7 @@ Resources are populated automatically for Bicep Recipes for any new Azure or AWS
 Recipes leverage [Bicep registries](https://learn.microsoft.com/azure/azure-resource-manager/bicep/private-module-registry) for template storage. Once you've authored a Recipe, you can publish it to your preferred OCI-compliant registry with [`rad bicep publish`]({{< ref rad_bicep_publish >}}):
 
 ```bash
-rad bicep publish --file myrecipe.bicep --target br:myregistry.azurecr.io/recipes/myrecipe:1.1.0
+rad bicep publish --file myrecipe.bicep --target br:myregistry.ghcr.io/recipes/myrecipe:1.1.0
```

Comment:
Need help checking that the target path for the command would still technically be valid here or if this requires `azurecr`.

Reply:
```suggestion
rad bicep publish --file myrecipe.bicep --target br:ghcr.io/USERNAME/recipes/myrecipe:1.1.0
```